build/export-openapi.sh

```diff
@@ -84,7 +84,7 @@ for f in *.json; do
   contents=$(cat "$f" | jq 'del(.description)' | sed 's/\\n/\\\\n/g' | sed '$d' | sed '1d' | sed ':a;N;$!ba;s/\n/\\n/g' | sed 's/\$/\\$/g' | sed 's/\&/\\&/g' | sed 's@\"@\\"@g')
   ref=$(basename "$f" .json)
 
-  find -maxdepth 1 -name '*.json' | xargs sed -i 's@"$ref": "#/definitions/'"$ref"'"@'"$contents"'@g'
+  find -maxdepth 1 -name '*.json' | awk -F, -v contents="$contents" '{ gsub("\"\\$ref\": \"#/definitions/" ref "\"", contents, $0)} {print}'
```

Comment:
What was wrong with sed 😃?

Reply:
sed complains, "Argument list too long"

Reply:
Now we're getting "./export-openapi.sh: line 87: /usr/bin/xargs: Argument list too long" 😬 so at least a different issue!

Reply:
In the `awk` command, why do we need `-F,`? We probably don't want print either.

Reply:
`-F` is to specify the field separator. Will remove `-F` and print.

Reply:
I understood what `-F` does but do we need it?

Reply:
Below two files modified using `make gen-embedded-openapi`  with the command `find -maxdepth 1 -name '*.json' | awk -v contents="$contents" '{ gsub("\"\\$ref\": \"#/definitions/" ref "\"", contents, $0)}'`

1) _io.k8s.api.core.v1.PodTemplateSpec.yaml - https://gist.github.com/Kalaiselvi84/400398322a50efcd1a0c00d2c44c4c19

2) _io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta.yaml -https://gist.github.com/Kalaiselvi84/0c5f582223e036eb864b9c453b7341d7

Reply:
> I understood what -F does but do we need it?
You're right, it's not necessary. Excluded it🙂